databricks-geococoding/README.md

```diff
@@ -0,0 +1,41 @@
+![Precisely](../Precisely_Logo.png)
+
+# Databricks Geocoding Sample
+This sample for Databricks demonstrates how to install, configure, and execute against the appropriate geocoding spark driver. 
+After cloning or downloading the repo you will have 2 html files to be imported into your workspace https://docs.databricks.com/notebooks/notebooks-manage.html.
+
+## Data
+This sample does not include the data, but does show how to integrate with the Precisely Data Experience.  Using your credentials you will be assured to be using the most recent data vintage.  Also this expidites the setting up of data on your cluster.
+
+* **Geocoding Reference Data**: USA-MasterLocationData, USA-TomTom. 
+
+## Setting up DataExperience integration
+  1) Collect your Data Experience API Key and Secret Key by visiting https://data.precisely.com/autodownload  You will need these for the Geocoding Installation Workspace.
```

Comment:
for the Geocoding Installation notebook.

Reply:
done